### PR TITLE
DR-1204 update ingest test to use file bulk array endpoint

### PR DESCRIPTION
**Pull request by RoriCremer**
Is there a way to get the ingest file size out of the array response?

### CODE REVIEW

File: datarepo-clienttests/src/main/java/testscripts/IngestFile.java [hunk 1]
```diff
@@ -77,23 +73,31 @@ public void userJourney(ApiClient apiClient) throws Exception {
...
-    logger.debug(
-        "Successfully ingested file: path = {}, id = {}, size = {}",
-        fileModel.getPath(),
-        fileModel.getFileId(),
-        fileModel.getSize());
+    BulkLoadArrayResultModel bulkLoadArrayResultModel =
+        DataRepoUtils.expectJobSuccess(repositoryApi, ingestFileJobResponse, BulkLoadArrayResultModel.class);
+    BulkLoadFileResultModel fileInfo = bulkLoadArrayResultModel.getLoadFileResults().get(0);
+
+    System.out.println(
```
Comment (marikomedlock) [inline datarepo-clienttests/src/main/java/testscripts/IngestFile.java]: Can you keep this using the logger.debug with {} format?

File: datarepo-clienttests/src/main/java/testscripts/IngestFile.java [hunk 2]
```diff
@@ -3,12 +3,8 @@
 import bio.terra.datarepo.api.RepositoryApi;
 import bio.terra.datarepo.api.ResourcesApi;
 import bio.terra.datarepo.client.ApiClient;
-import bio.terra.datarepo.model.BillingProfileModel;
-import bio.terra.datarepo.model.DatasetSummaryModel;
-import bio.terra.datarepo.model.DeleteResponseModel;
-import bio.terra.datarepo.model.FileLoadModel;
-import bio.terra.datarepo.model.FileModel;
-import bio.terra.datarepo.model.JobModel;
+import bio.terra.datarepo.model.*;
```
Comment (marikomedlock) [inline datarepo-clienttests/src/main/java/testscripts/IngestFile.java]: Nit: I think we have a team norm against * imports. I think you can turn this off in IntelliJ.
Screen Shot 2020-07-24 at 12 19 36 PM

**Review by marikomedlock — APPROVED** [review]
Thank you for taking care of this! Approved provided two small comments are addressed.

**Review by fboulnois — COMMENTED** [review]
I agree with Mariko's earlier comments, otherwise lgtm!

File: datarepo-clienttests/src/main/java/testscripts/IngestFile.java (hunk 2 above)
Comment (RoriCremer) [reply datarepo-clienttests/src/main/java/testscripts/IngestFile.java]: omg I keep missing this I'm an idiot

File: datarepo-clienttests/src/main/java/testscripts/IngestFile.java [hunk 3]
```diff
@@ -94,13 +89,15 @@ public void userJourney(ApiClient apiClient) throws Exception {
...
+    BulkLoadFileResultModel fileInfo = bulkLoadArrayResultModel.getLoadFileResults().get(0);
+
     logger.debug(
-        "Successfully ingested file: path = {}, id = {}, size = {}",
-        fileModel.getPath(),
-        fileModel.getFileId(),
-        fileModel.getSize());
+        "Successfully ingested file: path = {}, id = {}",
+        fileInfo.getSourcePath(),
+        fileInfo.getFileId());
```
Comment (fboulnois) [inline datarepo-clienttests/src/main/java/testscripts/IngestFile.java:107]: I think you mentioned before you were trying to get the size out. Just double checking if that's still a value you're interested in pulling out.

Comment (RoriCremer) [reply datarepo-clienttests/src/main/java/testscripts/IngestFile.java:107]: I just didn't like getting rid of metadata---but I dont see a way to pull this out in the new format

File: datarepo-clienttests/src/main/java/testscripts/IngestFile.java (hunk 1 above)
Comment (RoriCremer) [reply datarepo-clienttests/src/main/java/testscripts/IngestFile.java]: shouldn't it be logger.info to match the other ones?

Comment (marikomedlock) [reply datarepo-clienttests/src/main/java/testscripts/IngestFile.java]: Ah up to you. I usually log at debug level if it isn't relevant to the test outcome, but this is very subjective.

**Pull request merged**